int32                frameID                    
int32                navID                             
int32[640]           pObs          
int32                nObs